modules/tensor_mechanics/include/actions/TensorMechanicsAction.h

```diff
@@ -33,7 +33,7 @@ class TensorMechanicsAction : public TensorMechanicsActionBase
   virtual InputParameters getKernelParameters(std::string type);
 
   ///@{ displacement variables
-  std::vector<NonlinearVariableName> _displacements;
```

Comment:
Not good, because these names are potentially used to generate non-linear variables (never aux variables).

Reply:
Ah, well, I guess it just doesn't really matter.